job_definitions/publish_draft_services.yml

```diff
@@ -0,0 +1,74 @@
+{% for environment in ['preview', 'production'] %}
+- job:
+    name: "publish-draft-services-{{ environment }}"
+    display-name: "Publish draft services - {{ environment }}"
+    project-type: freestyle
+    disabled: true
+    description: |
+      This job runs the publish-draft-services script, for a 'pending' or 'standstill' framework.
+
+      For each draft service submitted by a successful supplier, a new 'live' service object is created via the API.
+      The draft service is updated with the new service ID. For G-Cloud only, any documents associated with the service
+      are then copied to a public S3 bucket.
+
+      Warning: this job takes approximately 10 hours to run for a G-Cloud framework with 36,000 draft services.
+
+    parameters:
+      - string:
+          name: FRAMEWORK_SLUG
+          description: "The framework slug to publish services for, e.g. 'g-cloud-11'."
+      - string:
+          name: AWS_PROFILE
+          default: "{% if environment == 'production' %}production{% else %}development{% endif %}"
+          description: "AWS profile to copy documents as"
+      - bool:
+          name: SKIP_DOCS_IF_PUBLISHED
+          default: false
+          description: "Don't copy documents if the draft service has already been published."
+      - bool:
+          name: DRY_RUN
+          default: false
+          description: "Dry run - do not set values in the database or copy documents"
+    publishers:
+      - trigger-parameterized-builds:
```

Comment:
Can we also have a Slack message when this job starts?

Reply:
I am a little wary of adding more noise to the channel, but if it will be useful for people then happy to add?

Reply:
My reasoning is that we're going to be running this job very rarely, so we're not going to be seeing the message very often, and it might be useful for people to get a Slack reminder that this is going on. Obviously we could just post a message ourselves though.

Reply:
Yeah, I think a human posting an `@here` is probably more visible than a slackbot. Unless I get really creative with the icon 😹